common/djangoapps/third_party_auth/models.py

```diff
@@ -468,6 +487,10 @@ def get_config(self):
         idp_class = get_saml_idp_class(self.identity_provider_type)
         return idp_class(self.idp_slug, **conf)
 
+    @property
+    def required_extra_attributes(self):
```

Comment:
@haikuginger I went back into it and realized the reason I needed this to be a property is because it depends on `self` for getting the value of `attr_country` (we need the URN).